packages/react-instantsearch/src/connectors/connectHits.test.js

```diff
@@ -3,22 +3,45 @@
 import connect from './connectHits.js';
 jest.mock('../core/createConnector');
 
-const {getProvidedProps, getSearchParameters} = connect;
+const {getSearchParameters} = connect;
 
 describe('connectHits', () => {
-  it('provides the current hits to the component', () => {
-    const hits = [{}];
-    const props = getProvidedProps(null, null, {results: {hits}});
-    expect(props).toEqual({hits});
-  });
+  describe('single index', () => {
+    const context = {context: {multiIndexContext: {targettedIndex: 'index'}}};
```

Comment:
Agreed we can nested it. But I also think about removing the multi-index index name from the the context and basically forward this as a prop for each multi index widget. What do you think? 

I am currently working on a manual autocomplete, and I noticed that in order to build the key navigation, somehow I need to have a way to know the hits index. Right now it could have mean using the context from a user side (that's not good), or pass the index name as a props. It's might be hard to understand without any material, so when I'll push the example, I'll show you. 

We could also keep the context, and only for the hits widget, have the index name forward as a props.